Makefile

```diff
@@ -1339,6 +1339,7 @@ UNIT_TEST_PROGRAMS += t-mem-pool
 UNIT_TEST_PROGRAMS += t-prio-queue
 UNIT_TEST_PROGRAMS += t-strbuf
 UNIT_TEST_PROGRAMS += t-trailer
+UNIT_TEST_PROGRAMS += t-reftable-basics
```

Comment:
The tests here are ordered alphabetically, so this should be moved two lines up.